### PR TITLE
Persist/delete locations in/from location list

### PR DESCRIPTION
**Pull request by borjanpopovic**
Added localStorage and a delete function for the location list.

Location entries persist after reloading the app.

Single locations can be deleted by clicking on the trash can(delete button).

### CODE REVIEW

File: src/Components/LocationCard.js [hunk 1]
```diff
@@ -32,8 +32,9 @@ import star from '../assets/map-pins/star.png';
 import taxi from '../assets/map-pins/taxi.png';
 import wc from '../assets/map-pins/wc.png';
 import trash from '../assets/trash.svg';
-import mapPin from '../lib/icons';
-export default function LocationCard({ location }) {
+/* import mapPin from '../lib/icons'; */
```
Comment (SabMueller) [inline src/Components/LocationCard.js:35]: It looks like you don't need this import, maybe you should consider deleting it? :)

File: src/Components/LocationCard.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<!-- ```jsx
+<LocationCard />
```
Comment (SabMueller) [inline src/Components/LocationCard.md:2]: Great that you've added the Styleguidist! ✨

File: src/App.js [hunk 3]
```diff
@@ -6,13 +6,26 @@ import FooterNavigation from './Components/FooterNavigation';
...
 
   function addLocations(location) {
     setLocations([...locations, location]);
   }
 
+  useEffect(() => {
+    saveToLocal('locations', locations);
+  }, [locations]);
+
+  function removeFromList(newLocationList) {
```
Comment (SabMueller) [inline src/App.js:21]: nice, clean function! I like it :)

**Review by SabMueller — APPROVED** [review]
Looks good to me! Nice job ✨

**Comment by borjanpopovic** [comment]
Thanks a lot for your review! :)  I'll delete all unused and commented elements!

**Pull request merged**